<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageCenter</class>
 <widget class="QWidget" name="MessageCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetMain" native="true">
     <property name="widgettpe" stdset="0">
      <string notr="true">main</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widgetTitle" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>44</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>44</height>
           </size>
          </property>
          <property name="widgettype" stdset="0">
           <string notr="true">errortitle</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>15</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="QLabel" name="lbIcon">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="labeltype" stdset="0">
                <string notr="true">msgtitleIcon</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Warning</string>
               </property>
               <property name="labeltype" stdset="0">
                <string notr="true">msgtitle</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnClose">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="buttontype" stdset="0">
                <string notr="true">close</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetMsg" native="true">
          <property name="widgettype" stdset="0">
           <string notr="true">errorbody</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QListWidget" name="listWidget">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>245</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widgetInfo" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QScrollArea" name="scrollArea">
                    <property name="minimumSize">
                     <size>
                      <width>336</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="horizontalScrollBarPolicy">
                     <enum>Qt::ScrollBarAlwaysOff</enum>
                    </property>
                    <property name="widgetResizable">
                     <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="scrollAreaWidgetContents">
                     <property name="geometry">
                      <rect>
                       <x>0</x>
                       <y>0</y>
                       <width>334</width>
                       <height>221</height>
                      </rect>
                     </property>
                     <property name="widgettype" stdset="0">
                      <string notr="true">errorbody</string>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_7">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>20</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_6">
                        <property name="spacing">
                         <number>15</number>
                        </property>
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_4">
                          <item>
                           <widget class="QLabel" name="label_3">
                            <property name="minimumSize">
                             <size>
                              <width>110</width>
                              <height>0</height>
                             </size>
                            </property>
                            <property name="maximumSize">
                             <size>
                              <width>110</width>
                              <height>16777215</height>
                             </size>
                            </property>
                            <property name="text">
                             <string>ErorCode :</string>
                            </property>
                            <property name="alignment">
                             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                            </property>
                            <property name="labeltype" stdset="0">
                             <string notr="true">pagetitle</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QLabel" name="lbCode">
                            <property name="text">
                             <string>TextLabel</string>
                            </property>
                            <property name="labeltype" stdset="0">
                             <string notr="true">errorinfo</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_5">
                          <item>
                           <widget class="QLabel" name="label_5">
                            <property name="minimumSize">
                             <size>
                              <width>110</width>
                              <height>0</height>
                             </size>
                            </property>
                            <property name="maximumSize">
                             <size>
                              <width>110</width>
                              <height>16777215</height>
                             </size>
                            </property>
                            <property name="text">
                             <string>details :</string>
                            </property>
                            <property name="alignment">
                             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                            </property>
                            <property name="labeltype" stdset="0">
                             <string>pagetitle</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QLabel" name="lbDetails">
                            <property name="text">
                             <string>TextLabel</string>
                            </property>
                            <property name="labeltype" stdset="0">
                             <string notr="true">errorinfo</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_6">
                          <item>
                           <widget class="QLabel" name="label_6">
                            <property name="minimumSize">
                             <size>
                              <width>110</width>
                              <height>0</height>
                             </size>
                            </property>
                            <property name="maximumSize">
                             <size>
                              <width>110</width>
                              <height>16777215</height>
                             </size>
                            </property>
                            <property name="text">
                             <string>solution :</string>
                            </property>
                            <property name="alignment">
                             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                            </property>
                            <property name="labeltype" stdset="0">
                             <string notr="true">pagetitle</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QLabel" name="lbSolution">
                            <property name="autoFillBackground">
                             <bool>true</bool>
                            </property>
                            <property name="text">
                             <string/>
                            </property>
                            <property name="scaledContents">
                             <bool>false</bool>
                            </property>
                            <property name="wordWrap">
                             <bool>true</bool>
                            </property>
                            <property name="labeltype" stdset="0">
                             <string notr="true">errorinfo</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                        <item>
                         <spacer name="verticalSpacer">
                          <property name="orientation">
                           <enum>Qt::Vertical</enum>
                          </property>
                          <property name="sizeHint" stdset="0">
                           <size>
                            <width>20</width>
                            <height>40</height>
                           </size>
                          </property>
                         </spacer>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </widget>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="lbDateTime">
                    <property name="text">
                     <string>2021-02-27 12:32:24</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="labeltype" stdset="0">
                     <string notr="true">errorinfo</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
